docs/design/features/globalization-icu.md

```diff
@@ -0,0 +1,131 @@
+# .NET Globalization ICU on all runtimes
+
+Author: [Santiago Fernandez Madero](https://github.com/safern)
+
+In .NET Core our globalization APIs behavior used different underlying libraries, in Unix, we used [International Components for Unicode (ICU)](http://site.icu-project.org/home) and in Windows, we used [National Language Support (NLS)](https://docs.microsoft.com/en-us/windows/win32/intl/national-language-support). This caused some behavior differences in a handful number of Globalization APIs:
+
+- Cultures and culture data
+- String casing
+- String sorting and searching
+- Sort keys
+- String Normalization
+- Internationalized Domain Names (IDN) support
+- Time Zone display name on Linux
+
```

Comment:
```suggestion
Starting with .NET 5, developers have more control over the underlying library utilized, enabling applications to avoid such differences across platforms.
```